startvesktop

```diff
@@ -3,25 +3,19 @@
 # Set FEATURES to what goes after --enable-features; for example, UseSkiaRenderer,WebRTCPipeWireCapturer,etc,etc will then get turned into --enable-features=UseSkiaRenderer,WebRTCPipeWireCapturer,etc,etc.
 FEATURES="UseSkiaRenderer"
 TMPDIR="$XDG_RUNTIME_DIR/app/${FLATPAK_ID:-dev.vencord.Vesktop}"
-declare -a FLAGS=(--enable-gpu-rasterization --enable-zero-copy --enable-gpu-compositing --enable-native-gpu-memory-buffers --enable-oop-rasterization --disable-gpu-blocklist)
+declare -a FLAGS=()
 
 if [[ $XDG_SESSION_TYPE == "wayland" ]]
 then
     FEATURES="${FEATURES},WebRTCPipeWireCapturer"
 
     WAYLAND_SOCKET=${WAYLAND_DISPLAY:-"wayland-0"}
 
-    echo "Using Wayland, enabling PipeWire for video capture."
-    echo "Screenshare may be broken on first start."
-    echo "Make sure you have the legacy screencapturer enabled."
-
-    FEATURES="${FEATURES},WebRTCPipeWireCapturer"
-
     if [[ -e "$XDG_RUNTIME_DIR/${WAYLAND_SOCKET}" ]]
     then
       echo "Wayland socket is available, running natively on Wayland."
       echo "To disable, remove the --socket=wayland permission."
-      FLAGS+=(--ozone-platform=wayland)
+      FLAGS+=(--ozone-platform-hint=wayland)
```

Comment:
this should be auto

we could also consider enabling wayland by default. afaik, electron 27 is finally stable on wayland